src/newusers.c

```diff
@@ -1092,15 +1090,13 @@ int main (int argc, char **argv)
 	while (fgets (buf, sizeof buf, stdin) != NULL) {
 		line++;
 		cp = strrchr (buf, '\n');
-		if (NULL != cp) {
+		if (cp == NULL && feof (stdin) == 0) {
+			fprintf (stderr, _("%s: line %d: line too long\n"),
+				 Prog, line);
+			fail_exit (EXIT_FAILURE);
+		}
+		if (cp == NULL) {
```

Comment:
Uh, is this check inverted, or am I flitting too much from project to project?

Reply:
D'oh!  I had an internal BSOD, it seems :)

Fixed.